backend/src/components/fundingAgreements.js

```diff
@@ -10,12 +10,13 @@ const { isEmpty } = require('lodash')
 async function getFundingAgreements(req, res) {
   try {
     const fundingAgreements = []
-    let operation = 'ofm_fundings?$select=ofm_fundingid,ofm_funding_number,ofm_declaration,ofm_start_date,ofm_end_date,_ofm_application_value,_ofm_facility_value,statuscode,statecode'
+    let operation =
+      'ofm_fundings?$select=ofm_fundingid,ofm_funding_number,ofm_declaration,ofm_start_date,ofm_end_date,_ofm_application_value,_ofm_facility_value,statuscode,statecode,ofm_version_number'
```

Comment:
Do we use "ofm_version_number" in frontend (because I don't see any mapping update)?

Reply:
if we don't need it for now, we can remove it from the select

Reply:
no we don't use it - We just require that field so we can preform the "order by" operation

Reply:
I think we can orderby without having it in select statement

Reply:
hmm you are right! I didn't think that was the case. Ok ok it's gone :)